agent/php_execute.c

```diff
@@ -1050,7 +1028,7 @@ static void nr_php_execute_metadata_metric(
   const char* function_name;
   const char* scope_name;
 
-#ifdef PHP7
+#if ZEND_MODULE_API_NO >= ZEND_7_0_X_API_NO /* PHP7+ */
   scope_name = metadata->scope ? ZSTR_VAL(metadata->scope) : NULL;
   function_name = metadata->function ? ZSTR_VAL(metadata->function) : NULL;
 #else
```

Comment:
Is it OK if scope_name or function_name are NULL for the `snprintf()` on line1039 or could that cause a exception depending on how `snprintf()` is implemented?

Reply:
It's okay, it handles the null.  That is legacy code.